### DISCUSSION TITLE
Thank you for having developed DJL and feedback/gift

### DISCUSSION THREAD

**Discussion by adriens** [deepjavalibrary/djl#444] [General]
Hi,

just a ticket to thank you for having developped this project. Thanks to it we could manage to implement a full ML service at the Office.

I've written a complete article on this topic, feel free to take a look at it:

https://www.linkedin.com/posts/adrien-sales_tensorflow-springboot-apis-activity-6745433991475789824-e_vW

Kind regards
Adrien

**Comment by zachgk** [comment]
You're welcome @adriens. We're glad DJL was able to help! Thanks for writing the blog post to share your neat AI meeting room management system

**Comment by adriens** [comment]
Is there a place where people could drop the prototypes or services they implemented thanks to DJL ?

**Reply by zachgk** [reply]
Yeah. We have a list of demos you can add it to at https://github.com/aws-samples/djl-demo/blob/master/README.md. This list is built into our docs site at http://docs.djl.ai/master/docs/demos/index.html
